.vib/spring-cloud-dataflow/goss/goss.yaml

```diff
@@ -0,0 +1,28 @@
+file:
+  /opt/bitnami/spring-cloud-dataflow/conf/application.yml:
+    exists: true
+    filetype: symlink
+    mode: "0777"
+    owner: root
+  /bitnami/spring-cloud-dataflow:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: root  
+  /.m2/repository/org/springframework/cloud/stream/app:
+    exists: true
+    filetype: directory
+    mode: "0644"
+    owner: root 
+command:
+  user-id-test:
+    exec: id -u
+    exit-status: 0
+    stdout:
+      - 1001
+    stderr: []
+  java-test:
+    exec: java -version
+    exit-status: 0
+    stdout:
+    stderr: []  
```

Comment:
There is not a lot to verify using Goss, since the app doesn't come with the CLI installed. 
In order to obtain the dataflow CLI, a separate container should be installed which is out of scope for community at the time being.